docs: mention supabase (#56676)
metabase/metabase: docs/databases/connections/postgresql.md

Metabase supports the oldest supported version of PostgreSQL through the latest stable version. See [PostgreSQL versions](https://www.postgresql.org/support/versioning/).

## Connect to Supabase

> You can select PostgreSQL to connect to your Supabase database. Refer to [Supabase documentation](https://supabase.com/docs/guides/database/metabase) for more details.

Reviewer: ```suggestion
To connect to a Supabase database, select PostgreSQL. For more details, check out the [Supabase docs](https://supabase.com/docs/guides/database/metabase).
```
